<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_usr</class>
 <widget class="QWidget" name="create_usr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建新用户</string>
  </property>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>10</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认创建</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_theme">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>创建用户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_usr">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_passowrd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_confirm">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_usr">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_confirm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
